modules/apple_rules_lint/0.2.0/presubmit.yml

```diff
@@ -0,0 +1,30 @@
+matrix:
+  platform:
+  - centos7
+  - debian10
+  - ubuntu2004
+  - macos
+  - windows
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
+    build_targets:
+    - '@apple_rules_lint//lint:docs'
```

Comment:
Stardoc is very broken with Bzlmod right now, it's better to exclude these targets for now.

Reply:
So I see :) Thank you for the hint!